cis/plugins/validation/schema.json

```diff
@@ -271,6 +271,13 @@
             "type": "string",
             "description": "Shirt size for this user. Format undefined."
         },
+        "tags": {
+            "type": "array",
+            "description": "General information about users tags. Tags prefixed by a provider name are provided by a specific tag origin.\nFor example `providername_l10n` is provided by `providername`.",
```

Comment:
Why not use something similar to groups? It might be confusing to have `mozillians_l10n` for tags and `nda` (without the prefix) for groups.

Reply:
both tags and groups are prefixed